content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -51,18 +51,14 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
       * `--feature-gates="IPv6DualStack=true"`
       * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
       * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
-      * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` defaults to /24 for IPv4 and /64 for IPv6
+      * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` defaults to /24 for IPv4 and /48 for IPv6
```

Comment:
The default is `/64`. If `--node-cidr-mask-size-ipv6` is not explicitly set, then the cluster cidr mask size should be >= `/48` as the diff between cluster cidr mask size and node cidr mask size can't be > 16.

Reply:
> The diff between cluster cidr mask size and node cidr mask size can't be > 16.

That sounds worth calling out. Is that documented on the website?